flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -63,8 +63,9 @@ under the License.
 						<goals>
 							<goal>shade</goal>
 						</goals>
-						<configuration combine.self="override">
```

Comment:
this removal is safe because it the override was plain unnecessary, and we get rid of an odd special case.